transport_nantes/topicblog/models.py

```diff
@@ -776,9 +776,21 @@ class TopicBlogEmailSendRecord(models.Model):
 
     """Represent the fact that we sent an email.
     """
+
+    class StatusChoices(models.TextChoices):
+        # This is the default state
+        PENDING = 'PENDING', "Pending"
+        # Toggled to this state when Amazon SES sends the email
+        SENT = 'SENT', "Sent"
+        # Toggled to this state if Amazon SES refuses to send the email
```

Comment:
It's not that SES refuses, it's that SES accepted the mail for sending and later reports that it was unable to do so for reasons quite out of its control.  This is quite different from refusing.

Reply:
Changed to 
`        # Toggle to this state if Amazon SES accepted the sending, but is unable to deliver the email`